### PR TITLE
Chat with ai

### PR DESCRIPTION
**Pull request by elcruzo**
Screenshot 2024-07-22 at 9 20 26 AM

### CODE REVIEW

File: Backend/utils/context.txt [hunk 1]
```diff
@@ -0,0 +1,77 @@
+BeatBridge Application Context:
```
Comment (ahdahl) [inline Backend/utils/context.txt:1]: So is this page needed to explain to llama what the different pages on the site do?

**Review by ahdahl — COMMENTED** [review]
Can users ask the chat bot any question or only ones related to beat bridge?

File: Backend/utils/context.txt (hunk 1 above)
Comment (elcruzo) [reply Backend/utils/context.txt:1]: Yes to give information about the app

**Comment by elcruzo** [comment]
> Can users ask the chat bot any question or only ones related to beat bridge?

Yes, users can ask the chatbot any question, not only ones related to Beat Bridge. However, if the question is outside the scope of the app, the bot will inform users that it is an AI assistant within the app but will still attempt to answer their question.

**Review by ahdahl — APPROVED** [review]
Very cool! Great job adding this stretch feature :)

**Pull request merged**